Google.PowerShell/Sql/GcSqlFlagsCmdlet.cs

```diff
@@ -25,8 +25,19 @@ namespace Google.PowerShell.Sql
     /// <para type="description">
     /// Lists all available database flags for instances.
     /// </para>
+    /// <example>
+    ///   <para>
+    ///   Gets a list of database flags available for instances.
+    ///   </para>
+    ///   <para><code>
+    ///     PS C:\> Get-GcSqlFlags
+    ///   </code></para>
+    ///   <br></br>
+    ///   <para>(If successful, the command returns a list of database flags available for instances)</para>
+    /// </example>
     /// </summary>
     [Cmdlet(VerbsCommon.Get, "GcSqlFlags")]
+    [OutputType(typeof(IList<Flag>))]
```

Comment:
I think this should just be typeof(Flag). We give WriteObject an IList<Flag>, but then we tell it to iterate through the elements.